src/components/atoms/Modal/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useRef } from 'react';
+
+import Styled from 'styled-components';
+
+export interface IProps {
+  children: React.ReactElement;
+  className?: string;
+  isOpen: boolean;
+  setIsOpen: (value: boolean) => void;
+}
+
+function Modal({ ...props }: IProps): React.ReactElement {
+  const { children, isOpen, setIsOpen } = props;
+  const Ref = useRef<HTMLDivElement>(null);
+  const closeHandler = (evt: any): void => {
+    if (isOpen && (!Ref.current || !Ref.current?.contains(evt.target))) {
```

Comment:
```suggestion
    if (isOpen && !Ref?.current?.contains(evt.target)) {
```
이렇게 정리하면 같은 코드이지 않을까요??